src/utils/clmfates_interfaceMod.F90

```diff
@@ -1054,7 +1054,7 @@ subroutine dynamics_driv(this, nc, bounds_clump,      &
 
             call ed_update_site(this%fates(nc)%sites(s), &
                   this%fates(nc)%bc_in(s), &
-                  this%fates(nc)%bc_out(s))
+                  this%fates(nc)%bc_out(s),.false.)
```

Comment:
I'd prefer seeing the .false. with a keyword argument in front of it, so I have some idea of what this is in regard to.

Reply:
I think all of this can be actually done with:
`hlm_is_restart` without passing any extra arguments.

Reply:
I was thinking about using that too @mvdebolskiy , it might work, but I wasn't sure if that flag stays on just during the restart sequence (its been a while since I've worked with it), or during the time-stepping following restarts.  If it also stays true during the latter, it won't work because we will be turning of all canopy_structure calls during the run.  But I agree, it would be nicer to use the flag if possible.

Reply:
@mvdebolskiy , I checked the code and hlm_is_restart will not work for our needs in this scenario.  That flag tells us if the run (as a whole) was restarted.  But, what we need to know is if the current call to ed_update_site was called during the restart setup sequence.  The time-stepping that follows will also be true for hlm_is_restart, on a restarted run.

Reply:
@ekluzek I added in the keywords

Reply:
Perfect